ui/js/models/error.js

```diff
@@ -43,8 +43,10 @@ treeherder.factory('ThTaskclusterErrors', ['localStorageService', function (loca
         */
         format: function (e) {
             if (e.code === 'AuthenticationFailed') {
-                let creds = localStorageService.get('taskcluster.credentials');
-                if (creds && creds.certificate && creds.certificate.expiry) {
+                const creds = localStorage.getItem('taskcluster.credentials');
+                const parsedCreds = creds && JSON.parse(creds);
+
+                if (parsedCreds && parsedCreds.certificate && parsedCreds.certificate.expiry) {
                     let expires = new Date(creds.certificate.expiry);
```

Comment:
Should this line now be `parsedCreds.certificate.expiry`?

Reply:
Neither.  This whole conditional should, I think, go away -- it leads to an error message about 3-day expiration that is no longer accurate.  Taskcluster-client-web will take care of getting new credentials if they are expired, so you are unlikely to get AuthenticationFailed unless something else is seriously broken (like tc-login's service credentials are misconfigured or the auth service has failed).

Reply:
Agreed. Will remove it.